klyshko-provisioner/kii-provisioner-run.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2022 - for information on the respective copyright owner
+# see the NOTICE file and/or the repository https://github.com/carbynestack/klyshko.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Fail, if any command fails
+set -e
+
+mkdir -p ~/.cs
+cat <<EOF > cs-config
```

Comment:
Doesn't this config file show us that a custom application that uses the castor-tuple-upload-client library might be better-suited for the task?

Though in that case be careful, I think the part about removing the MP-SPDZ file header was only added to the CLI.

Reply:
See https://github.com/carbynestack/klyshko/pull/1#discussion_r905869469.